client/src/actions/index.ts

```diff
@@ -104,17 +108,17 @@ function prefetchEmbeddings(annoMatrix: any) {
 /*
 Application bootstrap
 */
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-const doInitialDataLoad = () =>
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-  catchErrorsWrap(async (dispatch: any) => {
+const doInitialDataLoad = (): ((
+  dispatch: Dispatch,
+  getState: () => RootState
+) => void) =>
+  catchErrorsWrap(async (dispatch: Dispatch) => {
     dispatch({ type: "initial data load start" });
 
     try {
       const [config, schema] = await Promise.all([
         configFetch(dispatch),
-        // @ts-expect-error ts-migrate(2554) FIXME: Expected 0 arguments, but got 1.
-        schemaFetch(dispatch),
+        schemaFetch(),
```

Comment:
NOTE: this is runtime code change, but should be safe since `schemaFetch` doesn't seem to accept arguments 👌